src/modules/users/user-reducer.js

```diff
@@ -6,12 +6,12 @@ const initState = {
   users: []
 };
 
-const addUser = ({ users }, { user }) => ({
-  users: [...users, { ...user, id: users.length }]
+const usersLoaded = (_, { users }) => ({
+  users
```

Comment:
It seems like it's not necessary to use the old `state` but even if the state has only a single field, always copy the old state. I mean in a real app, it's very common to add other fields into a reducer later and you need to be prepared for that.